typescript/cli/src/rebalancer/config/Config.ts

```diff
@@ -35,18 +35,23 @@ const WeightedChainConfigSchema = BaseChainConfigSchema.extend({
     .transform((val) => BigInt(val)),
 });
 
-// Min amount strategy config schema
-const MinAmountChainConfigSchema = BaseChainConfigSchema.extend({
+// Absolute values, represents in token units the min and the target
+const MinAmountAbsoluteConfigSchema = BaseChainConfigSchema.extend({
   minAmount: z
     .string()
-    .or(z.number())
+    .or(z.number().gt(1))
     .transform((val) => BigInt(val)),
-  buffer: z
+  target: z
     .string()
-    .or(z.number())
-    .transform((val) => BigInt(val))
-    .default('0')
-    .optional(),
+    .or(z.number().gt(1))
+    .transform((val) => BigInt(val)),
+});
+
+// Relative values, with 0 = 0%, and 1 = 100%
+// It's relative to the total sum of collateral across al rebalanceable chains
+const MinAmountRelativeConfigSchema = BaseChainConfigSchema.extend({
+  minAmount: z.number().min(0).max(1),
+  target: z.number().min(0).max(1),
```

Comment:
Wont this have some sort of type collision with the absolute which uses BigInt instead?